integration/330_application_edge_test.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Test short connections between processes"
+
+WEAVE_DOCKER_ARGS=$ADD_HOST_ARGS weave_on $HOST1 launch
+scope_on $HOST1 launch
+weave_on $HOST1 run -d --name nginx nginx
+weave_on $HOST1 run -dti --name client alpine /bin/sh -c "while true; do \
+	nc nginx.weave.local 80 || true; \
+	sleep 1; \
+done"
```

Comment:
This isn't actually that short of a connection, as one it is connected the `nc` will hang for a while.